grc/core/blocks/_build.py

```diff
@@ -114,6 +116,8 @@ def add_param(**data):
         if param_id in params:
             raise Exception('Param id "{}" is not unique'.format(param_id))
 
+        _validate_option_attributes(param_data)
```

Comment:
I think you missed to forward `block_id`

Reply:
Fixed